packages/universal-components/src/RangeDatePicker/RenderMonth.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { format } from 'date-fns';
+import memoize from 'memoize-one';
+import isEqual from 'react-fast-compare';
+
+import { Text } from '../Text';
+import { StyleSheet } from '../PlatformStyleSheet';
+import RenderWeek from './RenderWeek';
+import { type MonthDate, getMonthMatrix } from './libs';
+
+const MONTH_NAME_FORMAT = 'MMMM';
+
+type Props = {|
+  +monthDate: MonthDate,
+  +onDayPress: Date => void,
+  +selectedDate: Date,
+|};
+
+export default class RenderMonth extends React.Component<Props> {
+  shouldComponentUpdate(nextProps: Props) {
+    const checkDate = props =>
```

Comment:
We can move this function outside of class.

Reply:
Very good point

Reply:
Done